### ISSUE TITLE
Not able to send sms for trigger Attendance Status

### ISSUE THREAD

**Issue by pavanpawar4591** [GibbonEdu/core#902]
Be sure to check out the How to report a bug section in our contributor docs if you haven't already. 
https://github.com/GibbonEdu/core/blob/master/docs/CONTRIBUTING.md#how-to-report-a-bug
Then, edit this template to tell us about the issue. Thanks!

**Describe the bug**
i am try to send sms for trigger "Attendance Status" but parents list is not able to see there is bug in code. email code is used for send sms .

**To Reproduce**
1. Go to Home > Messenger > New Message
2. select only sms
3. select trigger Attendance Status (make sure for particular date you register attendance  and phone number is added for parents and priority is 1 for parent )
4. in sent report Home > Messenger > Manage Messages > View Send Report you are not able to see any entry of parent because there is bug in code.

**Expected behavior**
need to list parent details in Home > Messenger > Manage Messages > View Send Report .

**Screenshots**

**Your System**
Include as many relevant details about the environment you encountered the bug in:
 - Gibbon version v17.0.00 and V 18.0.0.0]
 - Browser chrome,

**Additional context**
i found the root cause of issue in file 
./modules/Messenger/messenger_postProcess.php

```
 while ($rowSMS=$resultSMS->fetch()) {
									$countryCodeTemp = $countryCode;
	   								 if ($rowEmail["countryCode"]=="")
	   									 $countryCodeTemp = $rowEmail["countryCode"];
	   								 $report = reportAdd($report, $emailReceipt, $rowEmail['gibbonPersonID'], 'Attendance', $t, 'SMS', $countryCodeTemp.$rowEmail["phone"]);
                                }
                              }
                            } //END SMS
```
need to fix as

```
    while ($rowSMS=$resultSMS->fetch()) {
									$countryCodeTemp = $countryCode;
	   								 if ($rowSMS["countryCode"]=="")
	   									 $countryCodeTemp = $rowSMS["countryCode"];
	   								 $report = reportAdd($report, $emailReceipt, $rowSMS['gibbonPersonID'], 'Attendance', $t, 'SMS', $countryCodeTemp.$rowSMS["phone"]);
                                }
                              }
                            } //END SMS

```

need to fix one more place in some line before

**Comment by SKuipers** [comment]
Hi Pavan,

Thanks for catching this and including the suggested fix 😃 Are you interested in submitting a PR to contribute your fix to the core? If so let me know & give a shout with any questions, otherwise we'd be happy to include your fix in the next version.

Thanks!

**Comment by pavanpawar4591** [comment]
i will raise PR . please guide me.

**Comment by pavanpawar4591** [comment]
Raised PR. Please check

**Comment by pavanpawar4591** [comment]
Need to raise PR for V 19 ?

**Comment by SKuipers** [comment]
Hi Pavan! Oops, sorry about the wait 😬 your PR came in right as we were releasing v18, and shortly afterwards we headed off for summer break. I've updated the base branch for your PR to v19, no need to submit a new one. 

I've added a small comment to your PR to make a change, otherwise your fix looks good 👍  once we merge it in to v19 you'll be able to copy your changed back to your v18 install.

Thanks!
